dali/operators/math/expressions/arithmetic_test.cc

```diff
@@ -134,7 +134,7 @@ TEST(ArithmeticOpsTest, TreePropagationLayoutError) {
     ptr->Resize({{1}, {2}}, DALI_INT32);
   }
   in[0]->SetLayout(TensorLayout());
-  in[1]->SetLayout(TensorLayout("HW"));
+  in[1]->SetLayout(TensorLayout("DH"));
```

Comment:
Actually, we should test both cases - the old one should _not_ throw, but instead extend the layout to `"DHW"`.

Reply:
We have tests for that in python though.